generators/server/templates/src/main/java/package/web/rest/UserResource.java.ejs

```diff
@@ -31,6 +31,8 @@ import <%= packageName %>.repository.search.UserSearchRepository;
 import <%= packageName %>.security.AuthoritiesConstants;
 <%_ if (authenticationType !== 'oauth2') { _%>
 import <%= packageName %>.service.MailService;
+import org.springframework.data.domain.Sort;
```

Comment:
tests are ❌ because this import is misplaced

Reply:
not really :D. the private method (and the if statement) should not be applied when we choose Oauth2 (there si no password or resetKey with oauth2.